default.nix

```diff
@@ -15,6 +15,9 @@ with pkgs.haskell.lib;
 let
   iohk-ops-extra-runtime-deps = [
     pkgs.git pkgs.nix-prefetch-scripts compiler.yaml
+    pkgs.wget pkgs.awscli # for scripts/aws.hs
+    pkgs.nodejs           # for cardano-sl/scripts/js/genesis-hash.js
```

Comment:
This does now work with `io` alias? I'd assume we need to add it to a dependency list `libraryDepends` or similar

Reply:
Good question, I was contemplative and unsure when making this change!

Now that I think about it more -- this should work for the non-interactive version, because that's got its `wrapProgram` scripts generated.

It would, however fail for the `runhaskell` version -- only in case with `pkgs.nodejs`, when we're regenerating genesis.

Reply:
I guess I'll add all of those to `libraryHaskellDepends`, as it was implemented previously.

Reply:
Done.